src/utils/supabase.ts

```diff
@@ -841,3 +845,98 @@ export const getChannel = async (
     message: "Successfully fetched the channel!",
   };
 };
+
+export const getTutors = async (
+  query: TutorQuery
```

Comment:
For the sake of function consistency, let's just change the name of this parameter to `queryData`.